examples/cross-contract-high-level/README.md

```diff
@@ -50,10 +50,13 @@ near deploy --accountId=cross_contract --wasmFile=../examples/cross-contract-hig
 
 ### Deploying another contract
 Let's deploy another contract using `cross-contract`, factory-style.
+
 ```bash
-near call cross_contract deploy_status_message "{\"account_id\": \"status_message\", \"amount\":1000000000000000}" --accountId=test_near 
+near call cross_contract deploy_status_message "{\"account_id\": \"status_message\", \"amount\":"10000000000000000000000000"}" --accountId=cross_contract
```

Comment:
I believe the added quotes also need a backslack:
```suggestion
near call cross_contract deploy_status_message "{\"account_id\": \"status_message\", \"amount\":\"10000000000000000000000000\"}" --accountId=cross_contract
```

Reply:
Oops, thanks, I'm too used to having the whole JSON in single quotes